tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -358,6 +373,45 @@ def case_aggregates_by_month(df, dfs_status):
                                          "accepted_without_errors": accepted,
                                          "accepted_with_errors": cases_with_errors})
 
-    aggregate_data['rejected'] = ParserError.objects.filter(file=df).filter(case_number=None).count()
+    aggregate_data['rejected'] = ParserError.objects.filter(
+        file=df).filter(case_number=None).count()
 
     return aggregate_data
+
+
+def total_errors_by_month(df, dfs_status):
```

Comment:
move to `aggregators.py` once 2711 merges

Reply:
2711 merged earlier this morning, you can now pull in latest from develop